examples/webhook.py

```diff
@@ -0,0 +1,15 @@
+# This is an example for sending message to a channel using its webhook url.
+# You can use this to send messages to a channel without using the bot.
+
+import aiohttp
+import asyncio
+import nextcord
+
+async def send_to_webhook(url, content):
+    # Create a new HTTP session and use it to create webhook object
+    async with aiohttp.ClientSession() as session:
+        webhook = nextcord.Webhook.from_url(url, session=session)
+        await webhook.send(content)
+
+loop = asyncio.get_event_loop()
```

Comment:
This is not the preferred way of doing this in Python anymore. Instead It's recommended you use `asyncio.run()`.

i.e.
```py
asyncio.run(send_to_webhook("url", "Hello, world!"))
```